docs/source/guides/benchmark_transcoding.rst

```diff
@@ -61,7 +68,7 @@ Running a sample benchmark on a GTX 1060 with default profiles:
 
 ::
     
-    ./livepeer_bench -in bbb/source.m3u8 -nvidia 0
+    ./livepeer_bench -in bbb/source.m3u8 -nvidia 0 -segs 10
```

Comment:
Is `-segs 10` needed?

Reply:
Ah, not really. Had used that to get fewer lines of output but makes sense to keep the flag out of this line.